onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.cc

```diff
@@ -1156,8 +1155,8 @@ Status PoolOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder, const N
   const auto& op_type = node_unit.OpType();
 
   int32_t op_code;
-  bool is_qlinear_average_pool = op_type == "QLinearAveragePool";
-  bool is_average_pool = op_type == "AveragePool" || is_qlinear_average_pool;
+  bool is_quant_average_pool = IsQuantizedOp(node_unit);
```

Comment:
This might not be true if we add Maxpool support later, maybe change to `is_quant_pool` and 
```
is_average_pool = op_type == "AveragePool" || op_type == "QLinearAveragePool";
```